Solutions/Marain.Tenancy.Specs/Integration/Features/TenancyApi.feature

```diff
@@ -0,0 +1,50 @@
+﻿@perFeatureContainer
+@withTenancyClient
+@useTenancyFunction
+
+Feature: Tenancy Api
+	In order to use Marain Tenant services
+	As a developer
+	I want to be able to access the Tenancy Api.
+
+Scenario: Get the OpenApi definition for the Api
+	When I request the tenancy service endpoint '/swagger'
+	Then I receive a 'OK' response
+
+Scenario: Get a tenant that does not exist
+	When I request the tenant with Id 'thistenantdoesnotexist' from the API
+	Then I receive a 'NotFound' response
+
+Scenario: Create a tenant
+	When I use the API to create a new tenant
+	| ParentTenantId                   | Name |
+	| f26450ab1668784bb327951c8b08f347 | Test |
+	Then I receive a 'Created' response
+	And the response should contain a 'Location' header
+
+Scenario: Get the root tenant
+	When I request the tenant with Id 'f26450ab1668784bb327951c8b08f347' from the API
+	Then I receive an 'OK' response
+	And the response content should have a string property called 'name' with value 'Root'
+	And the response should not contain an 'Etag' header
+	And the response should contain a 'Cache-Control' header with value 'max-age=600'
```

Comment:
Since this value normally comes from config, how does it get into the function under test?

Reply:
That's why the build's broken :)

When running locally, we load the `local.settings.json` from the test project. When running as part of the build it comes from an environment variable that's not set yet.

Reply:
Doesn't that mean I should have seen some corresponding changes in at least one of the `local.settings.template.json` files?

Reply:
Good point, yes. Added.